files/views.py

```
@@ -496,7 +496,61 @@ def edit_media(request):
+            if media_update_info.get('updated'):
+                import os
+                import shutil
+                from files.helpers import rm_file
+
+                # Clean up old files and encodings
+                old_file_path = media_update_info.get('old_file_path')
+                if old_file_path and os.path.exists(old_file_path):
+                    try:
+                        rm_file(old_file_path)
+                    except Exception:
+                        pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)